### PR TITLE
feat:created js, perf:changed some lines html

### PR DESCRIPTION
**Pull request by TheBignes19**

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,199 @@
+// Aqui eu busco no HTMl as tags que eu preciso 
+const startGameButton = document.querySelector(".button-start-quiz") 
+const questionsContainer = document.querySelector(".question-container")
+const answersContainer = document.querySelector(".answers")
+const stopButton = document.querySelector(".button-stop")
+const confirmButton = document.querySelector(".button-two")
+const questionText = document.querySelector(".question")
+
+// Aqui
```
Comment (Yan-pg) [inline script.js]: remove comments

File: script.js [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+        confirmButton.classList.remove("confirmAnswer")
+    }
+
+    confirmButton.addEventListener("click", () => {
+        validAnswer()
+    })
+}
+
+function validAnswer() {
+
```
Comment (Yan-pg) [inline script.js]: remove space not used

File: script.js [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+    })
+    document.querySelectorAll(".answer").forEach (button =>{
+        button.setAttribute("disabled", true)
+    })
+}
+
+function stopQuiz() {
+    startGame()
+}
+
```
Comment (Yan-pg) [inline script.js:143]: vish

File: script.js [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+
+
+
+
+
+
+
+
+
```
Comment (Yan-pg) [inline script.js]: vish

File: script.js [hunk 5]
```diff
@@ -0,0 +1,199 @@
...
```
Comment (Yan-pg) [inline script.js]: vish

File: script.js [hunk 6]
```diff
@@ -0,0 +1,199 @@
...
```
Comment (Yan-pg) [inline script.js]: vish

File: script.js [hunk 7]
```diff
@@ -0,0 +1,199 @@
...
```
Comment (Yan-pg) [inline script.js]: vish

File: script.js [hunk 8]
```diff
@@ -0,0 +1,199 @@
...
```
Comment (Yan-pg) [inline script.js]: vish

File: script.js [hunk 9]
```diff
@@ -0,0 +1,199 @@
...
+}
+
+
+
+
+
+
+
+
+
```
Comment (Yan-pg) [inline script.js]: vish

File: script.js [hunk 10]
```diff
@@ -0,0 +1,199 @@
...
+    startGame()
+}
+
+
+
+
+
+
+
+
```
Comment (Yan-pg) [inline script.js]: vish

File: script.js [hunk 11]
```diff
@@ -0,0 +1,199 @@
...
+function stopQuiz() {
+    startGame()
+}
+
+
+
+
+
+
+
```
Comment (Yan-pg) [inline script.js]: vish

File: script.js [hunk 12]
```diff
@@ -0,0 +1,199 @@
...
+function stopQuiz() {
+    startGame()
+}
+
+
+
+
+
+
```
Comment (Yan-pg) [inline script.js]: vish

File: script.js [hunk 13]
```diff
@@ -0,0 +1,199 @@
...
+}
+
+function stopQuiz() {
+    startGame()
+}
+
+
+
+
+
```
Comment (Yan-pg) [inline script.js]: vish

File: script.js [hunk 14]
```diff
@@ -0,0 +1,199 @@
...
+    })
+}
+
+function stopQuiz() {
+    startGame()
+}
+
+
+
+
```
Comment (Yan-pg) [inline script.js]: vish

File: script.js [hunk 15]
```diff
@@ -0,0 +1,199 @@
...
+        button.setAttribute("disabled", true)
+    })
+}
+
+function stopQuiz() {
+    startGame()
+}
+
+
+
```
Comment (Yan-pg) [inline script.js]: vish

File: script.js [hunk 16]
```diff
@@ -0,0 +1,199 @@
...
+    document.querySelectorAll(".answer").forEach (button =>{
+        button.setAttribute("disabled", true)
+    })
+}
+
+function stopQuiz() {
+    startGame()
+}
+
+
```
Comment (Yan-pg) [inline script.js]: vish

**Review by yeaah-dev — APPROVED** [review]

**Pull request merged**